test/ray_julia_jll/buffer.jl

```diff
@@ -18,11 +22,10 @@ using .ray_julia_jll: LocalMemoryBuffer, Data, Size, OwnsData, IsPlasmaBuffer
         @test result === data
     end
 
-    # TODO: Using `sizeof` is probably wrong for `pointer_from_objref` as there is probably
-    # additional Julia type metadata not being accounted for. This may be why we see a
-    # segfault when trying to use `pointer_from_objref`.
-    # https://github.com/beacon-biosignals/Ray.jl/issues/55
     @testset "copied object reference" begin
+        # Using `sizeof` is probably wrong for `pointer_from_objref` as there is probably
+        # additional Julia type metadata not being accounted for. This may be why we see a
+        # segfault when trying to use `pointer_from_objref`.
```

Comment:
```suggestion
        # Using `sizeof` is probably wrong for `pointer_from_objref` as there is probably
        # additional Julia type metadata not being accounted for. This may be why we see a
        # segfault when trying to use `pointer_from_objref`.  However as noted above, no
        # one should be using this in practice, and it's only used here to test buffer
        # functionality separately from testing serialization.
```